spec/features/user/order_show_spec.rb

```diff
@@ -195,4 +195,33 @@
     expect(order_test.items.first.inventory).to eq(12)
     expect(order_test.items.last.inventory).to eq(3)
   end
+  
+  it 'when all item in an order have been fulfilled, the order status changes to packaged' do
+    user = User.create!(name: "Steve",
+                        address:"123 Main St.",
+                        city: "Fort Collins",
+                        state: "GA",
+                        zip: "66666",
+                        email: "chunky_lover@example.com",
+                        password: "123password")
+
+    visit "/login"
+    fill_in :Email, with: user.email
+    fill_in :Password, with: user.password
+    click_button "Login"
+    mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
+    meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+
+    tire = meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 9)
+    paper = mike.items.create(name: "Lined Paper", description: "Great for writing on!", price: 20, image: "https://cdn.vertex42.com/WordTemplates/images/printable-lined-paper-wide-ruled.png", inventory: 0)
+    pencil = mike.items.create(name: "Yellow Pencil", description: "You can write on paper with it!", price: 2, image: "https://images-na.ssl-images-amazon.com/images/I/31BlVr01izL._SX425_.jpg", inventory: 100)
+  
+    
+    order_1 = user.orders.create!(name: user.name, address: user.address, city: user.city, state: user.state, zip: user.zip)
+    ItemOrder.create!(item: tire, order: order_1, quantity: 3, price: tire.price, status: 1)
+    ItemOrder.create!(item: paper, order: order_1, quantity: 3, price: paper.price, status: 1)
+    order_1.status_to_packaged
+    visit "/profile/orders/#{order_1.id}"
+    expect(page).to have_content("Packaged")
```

Comment:
just wanted to check here that the method was working in the views even though the user controller won't use this method